platform/podjs/src/browserPod.ts

```diff
@@ -253,38 +256,35 @@ interface File {
  * It implements the PolyOut interface by storing files in IndexedDB
  * @class IDBPolyOut
  */
-export class IDBPolyOut implements PolyOut {
-    private async getFilesInfo(zipId: string): Promise<FileInfo[]> {
+class IDBPolyOut implements PolyOut {
+    private async getFileInfos(id: string): Promise<FileInfo[]> {
         const db = await openDatabase();
         return new Promise((resolve, reject) => {
             const tx = db.transaction([OBJECT_STORE_POLY_OUT], "readonly");
-            const request = tx.objectStore(OBJECT_STORE_POLY_OUT).get(zipId);
+            const store = tx.objectStore(OBJECT_STORE_POLY_OUT);
+            const request = store.index("id").getAll(id);
```

Comment:
Not in the original PR either; you're introducing an index here, `id`. Thus not exactly a "clean up", you're adding some behavior that has to be taken into account. And this is not simply an implementation detail, because that index is created elsewhere, in `openDatabase`. In order to avoid regressions, or simply change the behavior in both places in the same way, using a test might be more efficient than relying on someone having eyeballs in a specific PR where it's changed.

Reply:
Well, the original PR, removed the `keyPath` from the `poly-out` object store right here, and introduced unnecessary completely elsewhere (especially in `importArchive`) just to maintain and store arrays of files for each ID. Using an index is a much less intrusive (let alone simpler and more efficient) way to facilitate multiple archives per ID.

FWIW, this part didn't cause a regression in the original PR but was IMHO just poor design. But well, you cannot test agaist design ideoligies.